doc/releases/changelog-dev.md

```diff
@@ -20,109 +20,104 @@
 
 * A new function `qml.registers` has been added, enabling the creation of registers, which are implemented as a dictionary of `Wires` instances.
   [(#5957)](https://github.com/PennyLaneAI/pennylane/pull/5957)
+  [(#6102)](https://github.com/PennyLaneAI/pennylane/pull/6102)
 
 <h4>Quantum arithmetic operations 🧮</h4>
```

Comment:
Is this just waiting for the PRs to be merged by the end of the week?

Reply:
I spoke to Soran and he feels comfortable that they'll make it in :)